addons/mrp/models/mrp_production.py

```diff
@@ -373,14 +367,17 @@ def _compute_scrap_move_count(self):
     @api.depends('workorder_ids.date_planned_start', 'workorder_ids.date_planned_finished')
     def _compute_date_planned(self):
         for order in self:
-            date_planned_start_wo = date_planned_finished_wo = False
             if order.workorder_ids:
+                date_planned_start_wo = date_planned_finished = False
```

Comment:
date_planned_start_wo doesn't exist anymore so the reference should be removed i suppose